source/vmware_cluster_deployment/vmware_storage_setup/vcenter_datastores.rst

```diff
@@ -216,14 +216,19 @@ There's an alternative if you don't want to use SCHED_DS_REQUIREMENTS if you hav
 .. image:: /images/vcenter_datastore_storage_drs.png
     :align: center
 
+.. _vcenter_market:
+
 Marketplace with vCenter Datastores
 -----------------------------------
 
-The vCenter datastores are compatible with OpenNebula HTTP and S3 Marketplaces. A generic VM Template imported in OpenNebula is necessary: add the ID of the imported OpenNebula VM Template in ``/var/lib/one/remotes/etc/vmm/vcenter/vcenterrc`` in the default_template setting. More information about OpenNebula Marketplaces can be found :ref:`here <private_marketplace_overview>`.
+The vCenter datastores are compatible with OpenNebula HTTP and S3 Marketplaces. More information about OpenNebula Marketplaces can be found :ref:`here <private_marketplace_overview>`.
+
+.. note:: You can use a VM Template imported in OpenNebula: add the ID of the imported OpenNebula VM Template in ``/var/lib/one/remotes/etc/vmm/vcenter/vcenterrc`` in the default_template setting.
 
-.. important::
+The process of exporting an application from the marketplace is the following:
 
-    The vCenter VM Template used when downloading an image from a Marketplace needs to be empty (i.e without any disk or NIC attached).
+- If the datastore contains ``VCENTER_TEMPLATE_REF``, this will be used to create the OpenNebula template.
```

Comment:
we should explain the precedence, if i'm not mistaken it will be:

 * if you specify a VM Template ID in the CLI args or in Sunstone, that will be used
 * if a default is defined in vcenterrc, that will be used
 * if no VM Template ID is provided, if the DS contains VCENTER_TEMPLATE_REF, that will be used
 * otherwise, a new one will be created and added to the DS template